css/main.css

```diff
@@ -461,33 +461,144 @@ body {
   margin: 18px 0;
 }
 
-.error {
-  text-align: center;
-  color: red;
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
```

Comment:
- [ ] As requested by the previous reviewer, the project title and the close button seem to be hidden at the top and cannot be clicked on to close the modal. This happens because of two possible reasons: 
   1. when I am reviewing the project from 100% zoom level which is the default for most of your viewers and you were developing from a different zoom level lower than 100%
   2. You are developing from an external monitor which sometimes makes it difficult to speculate how the design would look like on a normal laptop screen size as from the monitor it would look okay but on the laptop screen it gets distorted

**For some context** I will include screenshots of at 100% zoom level and at 90%  to show you exactly what I mean.

**How it looks like at 100% zoom level**

[Screencast from 15-11-2022 08:34:43 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/201987772-f23c0d66-b963-4da4-ae39-66986ba2fdc4.webm)

**How it looks like at 90% zoom level:**

![image](https://user-images.githubusercontent.com/86318284/201987922-9dc077b6-9412-491e-8cd5-80d8dc073fde.png)

**Now** I hope this helps to clear the air and I hope the visuals and my explanation has somehow helped you to identify the cause and will help you address it.